### PR TITLE
The Road to DevOps Nirvana

### PR DESCRIPTION
**Pull request by Hamo69**
@temyers and @nickj69 this is very much a beginning. Please review and add bits.

### CODE REVIEW

File: _posts/2017-10-19-the-road-to-devops-nirvana.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+categories: devops enterprise transformation
+author: Hamish Tedeschi
+image: img/devops/the devops transformation helix.png
+---
+
+IT Transformation is not a phrase to be thrown around lightly. The mere mention of it will have some people shuddering at the thought of multi million dollar programmes attempting to shoehorn new organisational structures and / or technology into an organisation. And in many cases failing to achieve what it sets out to do, which was to fundamentally change the way an organisation operates.
+
+The mistake organisations make is that they view this change in people, process and technology as if it were a road to travel, with a destination. That couldn't be further from the truth. It may be a road to travel, but there is no destination. The destination is actually one of continuous improvement and a self sustaining loop. If this isn't understood from the beginning, then the ultimate goal will likely not be achieved.
+
+Ask yourself this: If I turned of your IT systems tomorrow, could your business still run effectively? I'd wager my house that you can't.
```
Comment (williamsia) [inline _posts/2017-10-19-the-road-to-devops-nirvana.markdown]: should be "turned off"

File: _posts/2017-10-19-the-road-to-devops-nirvana.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "The Road To DevOps Nirvana"
+date:   2017-10-19 02:03:58 +0000
+categories: devops enterprise transformation
+author: Hamish Tedeschi
+image: img/devops/the devops transformation helix.png
+---
+
+IT Transformation is not a phrase to be thrown around lightly. The mere mention of it will have some people shuddering at the thought of multi million dollar programmes attempting to shoehorn new organisational structures and / or technology into an organisation. And in many cases failing to achieve what it sets out to do, which was to fundamentally change the way an organisation operates.
```
Comment (temyers) [inline _posts/2017-10-19-the-road-to-devops-nirvana.markdown]: Feels like the article jumps between high level IT Transformation concepts and low level detail.  Who's the post aimed at?

Comment (Hamo69) [reply _posts/2017-10-19-the-road-to-devops-nirvana.markdown]: I have no idea. I think I was high when I wrote it.

File: _posts/2017-10-19-the-road-to-devops-nirvana.markdown (hunk 1 above)
Comment (Hamo69) [reply _posts/2017-10-19-the-road-to-devops-nirvana.markdown]: correct

**Comment by temyers** [comment]
@Hamo69 can we get this cleaned up and out?

Also the article date needs updating - to some time in the future

We can put future dates on the articles to front load content.

**Comment by Hamo69** [comment]
oh god. Yes, it needs something.
I need to think

**Comment by Hamo69** [comment]
@temyers shall I just kill this one?

**Comment by temyers** [comment]
I think there's value there - it just needs some work.

Plus - do we want to keep the Helix name or something else?

Probably aligned more closely with Systems Thinking and [The Three Ways](https://itrevolution.com/the-three-ways-principles-underpinning-devops/), rather than the low level detail (we've got it covered).

**Comment by Hamo69** [comment]
We basically want a sexy name for our delivery model..

**Comment by Hamo69** [comment]
Maybe it should be some sort of Rock. 

Like the Mech Rock Obsidian

**Pull request closed without merging**